### ISSUE TITLE
DHCP client doesn't accept multiple values for router option

### ISSUE THREAD

**Issue by juranki** [coreos/bugs#110]
I'm running a PXE cluster with a dnsmasq server.

When running 367.1.0, I was able to provide primary and backup default routes:
```
#default router
dhcp-option=3,N.N.N.1,N.N.N.2
```
After upgrading to 410.0.0, servers no longer got a default route, and communication didn't work beyond the subnet.

After some poking I noticed that servers get a default route if I drop the secondary route from dnsmasq configuration. But now there's no way to specify the backup route.

From [RFC 2132: DHCP Options and BOOTP Vendor Extensions](http://tools.ietf.org/html/rfc2132)
```
3.5. Router Option


   The router option specifies a list of IP addresses for routers on the
   client's subnet.  Routers SHOULD be listed in order of preference.

   The code for the router option is 3.  The minimum length for the
   router option is 4 octets, and the length MUST always be a multiple
   of 4.

    Code   Len         Address 1               Address 2
   +-----+-----+-----+-----+-----+-----+-----+-----+--
   |  3  |  n  |  a1 |  a2 |  a3 |  a4 |  a1 |  a2 |  ...
   +-----+-----+-----+-----+-----+-----+-----+-----+--
```

**Comment by crawford** [comment]
Sorry for letting this slip through the cracks. Can you reopen this if you are still having trouble?

**Comment by mischief** [comment]
this was partially fixed in systemd/systemd@a05185279bfc2420f60dc2a061d50bfcb646be70. it does not yet support multiple routes, but it will use the first router served by dhcp.
